clinica/pipelines/t1_linear/anat_linear_pipeline.py

```diff
@@ -273,6 +273,10 @@ def build_core_nodes(self):
         ants_registration_node.inputs.fixed_image = self.ref_template
         ants_registration_node.inputs.transform_type = "a"
         ants_registration_node.inputs.dimension = 3
+        if self.parameters.get("random_seed") is not None:
+            ants_registration_node.inputs.random_seed = self.parameters.get(
+                "random_seed"
+            )
```

Comment:
For our own records:

- This option was made available from version 2.3.0 onwards. It's probably old enough (June 2018) as of today.

- This option prevents execution with multiple threads. We are mostly interested in patient-level parallel processing so I don't think that's an issue.